cloud/deployment/deployment.go

```diff
@@ -22,7 +22,7 @@ import (
 
 var (
 	errInvalidDeployment    = errors.New("the Deployment specified was not found in this workspace. Your account or API Key may not have access to the deployment specified")
-	errInvalidDeploymentKey = errors.New("invalid Deployment selected")
+	ErrInvalidDeploymentKey = errors.New("invalid Deployment selected")
```

Comment:
any reason to make this public? I don't see the variable being used outside the `deployment` package

Reply:
`ErrInvalidDeploymentKey` is exported for testing convenience as  `workerqueue`  test uses it for an error path test.